app/ios/Plugin/AppPlugin.m

```diff
@@ -9,4 +9,5 @@
            CAP_PLUGIN_METHOD(getLaunchUrl, CAPPluginReturnPromise);
            CAP_PLUGIN_METHOD(getState, CAPPluginReturnPromise);
            CAP_PLUGIN_METHOD(removeAllListeners, CAPPluginReturnNone);
+           CAP_PLUGIN_METHOD(minimizeApp, CAPPluginReturnNone);
```

Comment:
can you move it up before removeAllListeners?

Reply:
And the return type should be `CAPPluginReturnPromise` since in the types you said it returns `Promise<void>;`